### ISSUE TITLE
Server does not send authetication data in connection request

### ISSUE THREAD

**Issue by akafortyseven** [zaidka/genieacs#251]
If I understand correctly when I do connection request like this
```
  curl -i 'http://localhost:7557/devices/D463FE-ABCABDD00042/tasks?connection_request' \
  -X POST \
  --data '{"name":"refreshObject", "objectName":""}'
```
GenieACS should take Device.ManagementServer.ConnectionRequestPassword and Device.ManagementServer.ConnectionRequestUsername, use it to create authentication data for digest authentication, attach this data to HTTP GET request and send it to client.
But this never happens in my case. GenieACS sends HTTP GET request without digest authentication data.  What to do?

**Comment by akcoder** [comment]
Do you have the credentials specified in genieacs/config/auth.js file?

"use strict";

function connectionRequest(deviceId, url, username, password, callback) {
  return callback("myUsername", "myPassword");
}

exports.connectionRequest = connectionRequest;

**Comment by akafortyseven** [comment]
> Do you have the credentials specified in genieacs/config/auth.js file?

Yes, I specified them and then restart server. I use data from Device.ManagementServer.ConnectionRequestUsername as "myUsername" and data from Device.ManagementServer.ConnectionRequestPassword as "myPassword".

**Comment by zaidka** [comment]
Can you confirm the device sends an authentication challenge response? If you can share some packet capture that would be great.

**Comment by nick9015** [comment]
Hi, I can put password to the ACS service so that not all devices can connect to my server, I can not find where to configure this. On my modem, I charge any username and password, and I can see the information. Thank you

**Comment by akcoder** [comment]
Please close this issue and take your request to the mailing list. GitHub is for defect tracking, not support requests. 

-dan
